app/src/main/java/com/nervousfish/nervousfish/activities/ContactActivity.java

```diff
@@ -0,0 +1,62 @@
+package com.nervousfish.nervousfish.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+
+import com.nervousfish.nervousfish.ConstantKeywords;
+import com.nervousfish.nervousfish.R;
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The ContactActivity shows the contacts information and his public keys.
+ */
+@SuppressWarnings("PMD.AtLeastOneConstructor")
+public final class ContactActivity extends AppCompatActivity {
+
+    private final List<String> keys = new ArrayList<>();
+    //In the future we will use this field more than once
+    @SuppressWarnings("PMD.SingularField")
```

Comment:
Could you put the comments (in all relevant classes) after the actual suppression for clarity 😄